pkg/server/commands/list_objects_test.go

```diff
@@ -46,14 +46,14 @@ func TestListObjectsDispatchCount(t *testing.T) {
 		{
 			name: "test_direct_relation",
 			model: `model
-			schema 1.1
+				schema 1.1
```

Comment:
This is really nested - can you put the `model` clause on a newline below and start everything inline from there. For example,

```go
model: `
model
  schema 1.1

type user
type document
  relations
    define viewer: [user]
 `,
```

Reply:
Thanks for your feedback @jon-whit 
I wrote a script for this. I'll adjust it to move the content after ` to be in a new line aligned at column 0

Here's a new example with the expected output
~~~
	model := testutils.MustTransformDSLToProtoWithID(`
model
schema 1.1

type user

type document
  relations
    define viewer: [user, document#viewer] or editor
    define editor: [user, document#viewer]`)
~~~

Once approved, I can take care of other packages by just re-running the script